src/components/cesium_map/api/server.js

```diff
@@ -167,11 +166,10 @@ async function pointStream(query, perdoc_cb = null, finaldoc_cb = null, error_cb
           let jsonString = buffer.substring(0,i+1);
           pointsArr.push(jsonString);
           //done reading one json obj, discard current object from buffer 
-          if (buffer[i+1] === ",") {
-            buffer = buffer.substring(i+2); // discard comma 
-          }
-          else {
-            buffer = buffer.substring(i + 1);
+          buffer = buffer.substring(i); // continue on next part of string
+          let commaIdx = buffer.indexOf(",");
```

Comment:
Based on server, some return whitespaces in between the comma so change this logic.